region/norfair/west/Crocomire Escape.json

```diff
@@ -254,6 +254,192 @@
         "Samus will not be visible during the climb."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Setup - Get Hit By Geruta, Heatproof, Morph",
+      "requires": [
+        "h_heatProof",
+        "canBePatient",
+        "canCameraManip",
+        "Morph",
+        {"or": [
+          "canTrickyDodgeEnemies",
+          "SpaceJump",
+          "Charge",
+          "Ice",
+          "Plasma",
+          {"ammo": {"type": "Missile", "count": 2}},
+          {"ammo": {"type": "Super", "count": 2}},
+          {"ammo": {"type": "PowerBomb","count": 2}},
+          {"enemyDamage": {
+            "enemy": "Dragon",
+            "type": "fireball",
+            "hits": 5
+          }},
+          {"and": [
+            "h_lavaProof",
+            {"and": [
+              "ScrewAttack",
+              "h_canBombThings"
+            ]}
+          ]}
+        ]},
+        {"or": [
+          "SpaceJump",
+          {"and": [
+            "HiJump",
+            "SpeedBooster",
+            "canWalljump"
+          ]},
+          {"and": [
+            "HiJump",
+            "canSpringBallJumpMidAir"
+          ]},
+          {"and": [
+            "canBeVeryPatient",
+            "canJumpIntoIBJ"
+          ]},
+          {"and": [
+            "canBeVeryPatient",
+            "Grapple",
+            {"or": [
+              "HiJump",
+              "canSpringBallJumpMidAir"
+            ]}
+          ]}
+        ]},
+        {"or": [
+          {"ammo": {"type": "Super", "count": 1}},
+          {"obstaclesCleared": ["A"]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithGModeSetup": {}
+      },
+      "collectsItems": [3],
+      "unlocksDoors": [
+        {
+          "types": ["ammo"],
+          "requires": []
+        }
+      ],
+      "flashSuitChecked": true,
+      "note": "Lure the Geruta to the left door. This may require many jumps to get it to swoop up onto the ledge and not get stuck.",
+      "devNote": "It's possible to avoid the item, but the Geruta will get stuck on it and usually require many swoops to get past it."
+    },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Setup - Get Hit By Geruta, Heatproof, Morphless",
+      "requires": [
+        "h_heatProof",
+        "canBePatient",
+        "canCameraManip",
+        {"or": [
+          "canInsaneJump",
+          "Ice",
+          "h_lavaProof",
+          {"enemyDamage": {
+            "enemy": "Geruta",
+            "type": "contact",
+            "hits": 1
+          }},
+          {"and": [
+            "canNeutralDamageBoost",
```

Comment:
I'm not completely sure how this works.  I think the damage boost is for difficulty placement.  And avoiding a second(the first) fireball takes a room entry that is a higher difficulty - or its assumed you kill the other dragon while in the lava.